testsuite/httpx/__init__.py

```diff
@@ -103,3 +103,11 @@ def request(
             if len(e.args) > 0 and any("Name or service not known" in arg for arg in e.args):
                 raise UnexpectedResponse("Didn't expect 'Name or service not known' error", None) from e
             raise
+
+    def get_many(self, url, count, *, params=None, headers=None, auth=None) -> list[Response]:
```

Comment:
What if this method returned only `status_codes`? It would make checking it in the test easier

Reply:
Yes, but I want to mimic the behavior of `get` method. In addtion, we are used to call `.status_code` in our asserts.